packages/client/src/app/components/work-card/work-card.component.ts

```diff
@@ -0,0 +1,32 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+import * as v from 'voca';
+import { ContentKind } from '@pulp-fiction/models/content';
+
+import { calculateApprovalRating } from '../../util/functions';
+
+@Component({
+    selector: 'work-card',
+    templateUrl: './work-card.component.html',
+    styleUrls: ['./work-card.component.less']
+})
+export class WorkCardComponent implements OnInit {
+    @Input() content: any;
```

Comment:
huh, can we not make this a more specific type? coulda sworn we had a base Content interface